### ISSUE TITLE
Trap issue on using of constrain

### ISSUE THREAD

**Issue by orkhanalizade** [robb/Cartography#176]
Hi. Why I get trap error in XCode on lines:

    constrain(swipeableView, view) { view1, view2 in
            view1.left == view2.left+50
            view1.right == view2.right-50
            view1.top == view2.top + 120
            view1.bottom == view2.bottom - 100
        }

this line `constrain(swipeableView, view) { view1, view2 in` crashes my app with `trap` error. Do you know how can I fix it?

**Comment by robb** [comment]
Is either of `swipeableView` or `view` a reference of type `UIView!` and `nil`, by chance?

**Comment by orkhanalizade** [comment]
@robb no. You can look at my SO question here http://stackoverflow.com/questions/33057235/strange-trap-error-in-xcode-swift?noredirect=1#comment53935032_33057235

**Comment by orkhanalizade** [comment]
@robb any news?

**Comment by robb** [comment]
Not sure this is actually Cartography related, googling `fastBindLazySymbol` turns up multiple similar errors with other libraries. Could it be a Cocoapods/setup problem [as it was here](https://github.com/Alamofire/Alamofire/issues/556#issuecomment-123717198)?

**Comment by orkhanalizade** [comment]
@robb this error appears 2-3 days ago. Before it all was good! So strange

**Comment by robb** [comment]
Nothing really changed with Cartography in that timeframe.
